parachains/runtimes/assets/statemint/src/lib.rs

```diff
@@ -531,12 +531,23 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = weights::pallet_collator_selection::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const UseUserConfiguration: bool = false;
```

Comment:
What does this do exactly?

Reply:
you can check this PR https://github.com/paritytech/substrate/pull/12087/files#diff-4b420db30e4fc16308ea5d9858a9e56e1895d670f903a76fde733e316125b2a1